react/src/components/ExactMatchFilterToggle.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useEffect, useState } from "react";
+import { Column } from "@material-table/core";
+import { Checkbox, InputAdornment, makeStyles, TextField, Tooltip } from "@material-ui/core";
+import { FilterList } from "@material-ui/icons";
+import { updateSearchTypeAndRequery } from "../functions";
+import { Dataset, Participant } from "../typings";
+
+interface ExactMatchFilterToggleProps {
+    MTRef: React.MutableRefObject<any>;
+    columnDef: Column<Participant> | Column<Dataset>;
+    onFilterChanged: (rowId: string, value: any) => void;
+}
+
+const useStyles = makeStyles(() => ({
+    input: {
+        "& .MuiInputBase-input": {
+            width: 30,
+            "&:focus": {
+                width: 120,
+                transition: "width 0.35s ease-in-out",
+            },
+        },
+    },
+}));
+
+export default function ExactMatchFilterToggle(props: ExactMatchFilterToggleProps) {
+    const classes = useStyles();
+    const [exactMatch, setExactMatch] = useState<boolean>(false);
+
+    useEffect(() => {
+        if (props.columnDef.field) {
+            updateSearchTypeAndRequery(props.MTRef, {
+                column: props.columnDef.field,
+                exact: exactMatch,
+            });
+        }
+    }, [props.MTRef, props.columnDef.field, exactMatch]);
+
+    return (
+        <TextField
+            autoFocus
```

Comment:
This component is used multiple times on the page, so only the last one actually receives focus. I think we don't need this.

Reply:
Oh that's right. Forgot to delete this one. Will do - thanks!